### PR TITLE
Cap'n Pester hails reveal commonly missed shortcuts

### PR DESCRIPTION
**Pull request by tehhowch**
Ref #2628 

Additions and corrections welcomed.

### CODE REVIEW

File: data/persons.txt [hunk 1]
```diff
@@ -111,6 +111,18 @@ person "Cap'n Pester"
 			"new weapons or other outfits."
 			"improved planet descriptions."
 			"more varied hail messages."
+	phrase
+		word
+			`Press "F" on the map screen to search for systems and planets.`
+			`You can open the map screen when being offered a mission.`
+			`Press "Alt" to highlight shortcut keys.`
+			`"Shift+S" will sell all your minerals and commodities on the Trade screen, or outfits in cargo if you have nothing else to sell.`
+			`Press "Shift+[number]" to add your selected escort to a hotkey, and then select by pressing that number again.`
```
Comment (Bladewood) [inline data/persons.txt]: I would suggest this as a rewrite:
> Press "Shift+[number]" to bind selected escorts to a hotkey, and you can then select them by pressing that number key again.

File: data/persons.txt [hunk 2]
```diff
@@ -111,6 +111,18 @@ person "Cap'n Pester"
...
+			`Pressing "Shift+T" will hail the planet you wanted to land on, even if you're fighting a ship.`
+			`Pressing "I" at the Outfitter will install that outfit from your cargo hold.`
+			`Use "Shift" and "Ctrl" to select multiple ships in the Shipyard and Outfitter.`
```
Comment (Bladewood) [inline data/persons.txt]: Capitalization on Outfitter and Shipyard?

**Comment by Bladewood** [comment]
I think this is way too explicit, even for Captain Pester.

File: data/persons.txt (hunk 2 above)
Comment (tehhowch) [inline data/persons.txt]: The names of menus should be capitalized:
"File -> Settings" vs "file -> settings"

**Comment by MessyMix** [comment]
I actually think this is a great idea. Some helpful easter eggs, and really gives a point to Cap'n Pester besides being a vestige of EV. :)

File: data/persons.txt (hunk 1 above)
Comment (Amazinite) [inline data/persons.txt]: It's ctrl+[number], not shift+[number].

**Comment by tehhowch** [comment]
While the shortcuts have been added to the Player's Manual, it may still be valuable to have some ship in the game provide these hails as not all players will search for a manual

**Comment by Amazinite** [comment]
Why not have a section in preferences that shows all shortcuts? That way all shortcuts are readily available in game without having to find and pester Pester for hails.

**Comment by tehhowch** [comment]
It can definitely be a long term goal to have all the hotkeys displayed in some game-accessible location (and also possibly user configurable). An infrequent visit by an already 4th-wall breaking ship is a good start until then

**Pull request closed without merging**